customersupport/templates/new-ticket-modal.html

```diff
@@ -0,0 +1,63 @@
+<div class="modal fade" id="newTicketModal" tabindex="-1" role="dialog" aria-labelledby="newTicketModalLabel">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span
+                        aria-hidden="true">&times;</span></button>
+                <h4 class="modal-title" id="newTicketModalLabel">New Ticket</h4>
+            </div>
+            <div class="modal-body">
+                <form>
+                    <div class="row">
+                        <div class="col-sm-12">
+                            <div class="form-group">
+                                <label for="titleForm">Ticket Name</label>
+                                <input type="text" class="form-control" id="titleForm" value="">
+                            </div>
+                        </div>
+                    </div>
+
+                    <div class="row">
+                        <div class="col-sm-12">
+                            <div class="form-group">
+                                <label for="status">Status</label>
+                                <select class="form-control" id="status">
+                                    <option>New</option>
+                                    <option>Pending</option>
+                                    <option>Closed - Successful</option>
+                                    <option>Closed - Failure</option>
+                                </select>
+                            </div>
+                        </div>
+                    </div>
+
+                    <div class="row">
+                        <div class="col-sm-12">
+                            <div class="form-group">
```

Comment:
I know I suggested ignoring this so the page made sense in general, but I want to note that we need to make this work like an actual form.
This PR is fine, but that's the next task to fix here.